internal/controller/defaults.go

```diff
@@ -65,25 +64,24 @@ var defaultDeploymentStrategy = appsv1.DeploymentStrategy{
 	},
 }
 
-var operatorNamespace = utils.Call(func() string {
-	namespace := os.Getenv("OPERATOR_NAMESPACE")
-	if namespace == "" {
+var (
+	operatorNamespace    string
+	operatorConfigName   string
+	serviceAccountPrefix string
+)
+
+func InitConfig() {
+	if operatorNamespace := os.Getenv("OPERATOR_NAMESPACE"); operatorNamespace == "" {
```

Comment:
Isn't the walrus shawdowing global operatorNamespace var here?

Reply:
Fixed, thanks!